### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by nikimacm**
# Zero To Mastery - Webblocks 2022 📦📦

Thank you for creating a pull request to the Web Blocks 2022 project 🙏 <br>
Please provide the following details below so your pull request can be processed more efficiently.

## WHAT THIS PULL REQUEST INTRODUCES

Please provide detailed information here about what your pull request introduces, please be as descriptive as possible 🤜

> My pull request introduces a halloweeny word counter. Type word in the form provided and see the amount underneath.

## ABOUT THE ASSETS IN THIS PROJECT

Please provide the copyright status of any images or media included in your block. The more information you can provide the more likely your PR will be merged quickly and efficiently.  

##Credits

Image: Photo by Ellie Burgin: https://www.pexels.com/photo/halloween-flat-lay-3177260/
Code: I watched a tutorial by [gilgeekify programming] (https://www.youtube.com/channel/UCdKndoYGm3wufSskFcjVHdQ) to learn how to create the web app.

Go over all the following points, and put an `x` in all the boxes that apply.

- [ ] There are no images or media assets in this PR
- [ ] I own the intellectual rights to the assets in this PR
- [x ] I utilised a free use media site such as `Unsplash` or `Pexels`
- [ ] I am unsure

## PULL REQUEST CHECKLIST

Please confirm that your pull request contains all the required information that we need to merge your pull request. <br />
⚠ **We are likely unable to merge any PRs that can't check all of these boxes**
<br /><br />
Go over all the following points, and put an `x` in all the boxes that apply.

- [ x] I have created my block in my own directory
- [x ] I have made sure to include a meta.json file inside my directory that contains the correct information about **MY** block
- [x ] I have made sure to include a screenshot of my block and named it `icon.png` inside of my directory
- [x ] I have **NOT** edited `src/blocks.js`, `package.json`, `readme.md`, `package-lock.json` or any other file outside of my directory
- [ x] I haven't used any copyrighted materials and have documented where the assets came from in this PR

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (MattCSmith) [inline .gitpod.yml:1]: This file needs removing

**Review by MattCSmith — CHANGES_REQUESTED** [review]
The .gitpod.yml and package-lock.json should not be committed. Please remove this files

**Comment by MattCSmith** [comment]
Hey, I can't currently merge this pull request with the two files mentioned above. Please removed them so we can proceed with merging. 

Since its unlikely that this PR will be merged by the Hacktoberfest deadline which is rapidly approaching I will go ahead and and the hacktoberfest-accepted label to this PR so you are still credited for it.

**Comment by nikimacm** [comment]
Hi Matt,

Thank you so much. I have deleted the files from both branches.
All the best
Nicola

**Comment by nikimacm** [comment]
Hi Matt,

Thank you so much. I have deleted the files from both branches.
All the best
Nicola

**Pull request merged**